src/pathfinding_service/api.py

```diff
@@ -289,6 +289,20 @@ def get(  # pylint: disable=no-self-use
         return dict(request_count=request_count, responses=responses), 200
 
 
+class DebugEndpointIoU(PathfinderResource):
+    def get(self, source_address: Address) -> Tuple[dict, int]:  # pylint: disable=no-self-use
+        response = []
+        ious_sender = self.pathfinding_service.database.conn.execute(
+            "select * from iou where sender = ?", [source_address]
```

Comment:
We usually upper case sql keywords.

Reply:
Oh, and can't you use `get_iou` from the database class?

Reply:
`"select * from iou where SENDER = ?", [source_address]` ?

Reply:
> Oh, and can't you use get_iou from the database class?
nope, because I want to iterate over several IOUs 

```    
for iou in ious_sender:
TypeError: 'IOU' object is not iterable
```